server/controllers/profile.js

```diff
@@ -0,0 +1,56 @@
+const Profile = require("../models/Profile");
+const asyncHandler = require("express-async-handler");
+
+// @route GET /profiles/
+// @desc Returns a list of all profiles
+// @access Public
+exports.getAllProfiles = asyncHandler(async (req, res, next) => {
+  const profiles = await Profile.find({});
+  res.status(200).json(profiles);
+});
+
+// @route GET /profiles/:id
+// @desc Given an ID, return profile with that ID
+// @access Public
+exports.getProfile = asyncHandler(async (req, res, next) => {
+  try {
+    const profile = await Profile.findById(req.params.id)
+      .populate("review")
+      .populate("appointment");
+    res.status(200).json(profile);
+  } catch (e) {
+    res.status(404).json("No profile with that id was found");
+  }
+});
+
+// @route POST /profiles/create
+// @desc  Given parameters passed in, create a profile.
+// @access Private
+exports.createProfile = asyncHandler(async (req, res, next) => {
+  const profile = new Profile({ user: req.user._id, ...req.body });
+
+  try {
+    const newProfile = await profile.save();
+    res.status(201).json(newProfile);
+  } catch (e) {
+    console.log(e);
```

Comment:
Please watch for unnecessary logs to the console. Going forward, we should be cleaning up our logs done during development. Thanks!